website/docs/r/container_registry_docker_credentials.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: "digitalocean"
+page_title: "DigitalOcean: digitalocean_container_registry_docker_credentials"
+sidebar_current: "docs-do-resource-container-registry-docker-credentials"
+description: |-
+  Get DigitalOcean Container Registry docker credentials
+---
+
+# digitalocean_container_registry_docker_credentials
+
+Get docker credentials for your DigitalOcean container registry.
+
+An error is triggered if the provided container registry name does not exist.
+
+## Example Usage
+
+### Basic Example
+
+Get the container registry:
+
+```hcl
+resource "digitalocean_container_registry_docker_credentials" "example" {
+  name = "example"
+}
+```
+
+### Docker Provider Example
+
+Use the `endpoint` and `docker_credentials` with the Docker provider:
+
+```hcl
+data "digitalocean_container_registry" "example" {
+  name = "example"
+}
+
+resource "digitalocean_container_registry_docker_credentials" "example" {
+  name = "example"
+}
+
+provider "docker" {
+  host = "tcp:localhost:2376"
```

Comment:
Testing this, I get:

> Error initializing Docker client: unable to parse docker host `tcp:127.0.0.1:2376/`

The Docker provider docs use: `tcp://localhost:2376` which gets past that error. On Ubuntu, I actually needed: `unix:///var/run/docker.sock`